_posts/en/2021-05-18-problems-of-project-based-funding.md

```diff
@@ -104,7 +104,7 @@ In fact, a Microsoft research showed that only about 1/3 of well-designed, well-
 
 ## Moving to a different software investment management method
 
-Since we expect moving to Digital will mean business operations will be done with software, it warrants us to review how we fund software investments so to enable faster feedback loop between policy makers and end-users.
+Since we expect moving to Digital means business operations to be done with software, it warrants us to review how we fund software investments so to enable faster feedback loop between policy makers and end-users.
```

Comment:
Hmm, I think this should be either "....Digital means business operations will be done with software"  Or "...Digital means business operations is done with software"  

Since 'moving to Digital' expresses  something in the future, I'm leaning towards "....Digital means business operations will be done ....."